src/passes.rs

```diff
@@ -1040,16 +1046,25 @@ impl<T: PassManagerSubType> PassManager<T> {
         }
     }
 
-    #[llvm_versions(7.0 => latest)]
+    #[llvm_versions(7.0)]
```

Comment:
Was this removed in 8.0?

Reply:
It was moved apparently, from the `llvm_sys::transforms::scalar` module to the `llvm_sys::transforms::aggressive_instcombine` module.

Reply:
Ohh I see.